tox.ini

```
@@ -31,11 +31,16 @@ deps =
   wheel
 commands =
   # safety measure to assure we do not accidentaly run tests with broken dependencies
+  # We add coverage options but not making them mandatory as we do not want to force
+  # pytest users to run coverage when they just want to run a single test with `pytest -k test`
   python -m pip check
   {envpython} -m pytest \
-  --cov "{envsitepackagesdir}/ansiblelint" \
   --junitxml "{toxworkdir}/junit.{envname}.xml" \
-  {posargs:}
+  {posargs:\
+    --cov "{envsitepackagesdir}/ansiblelint" \
```

Don't forget to keep the value from the config:
```suggestion
    --cov ansiblelint \
    --cov "{envsitepackagesdir}/ansiblelint" \
```